core/unit_test/TestTeam.hpp

```diff
@@ -410,7 +410,7 @@ class TestReduceTeam {
       team_exec = policy_type(league_size, team_size);
 
       for (unsigned i = 0; i < Repeat; ++i) {
-        result_type tmp(&result[i][0]);
+        result_type tmp(&result[i][0], 3);
```

Comment:
Should we create a `constexpr int size=3` and use it in `ScalarType result[Repeat][size]` and `result_type tmp(&result[i][0], size)`?

Probably isn't too important because it is a unit test, but could avoid confusion if the test is ever updated. Feel free to ignore.